docs/topics/proc_creating-a-reactive-http-service-with-webflux.adoc

```diff
@@ -0,0 +1,83 @@
+[id='creating-a-reactive-spring-boot-http-service-with-webflux_{context}']
+= Creating a reactive {SpringBoot} HTTP service with WebFlux
+
+Create a basic reactive Hello World HTTP web service using {SpringBoot} and WebFlux.
+
+.Prerequisites
+
+* Java JDK 8 or Java JDK 11 installed
+* Maven installed
+// this configuration might not apply entirely while examples still in tech preview
```

Comment:
Mention this in the doc?

Reply:
According to Libor, the reactive webflux starters are going to be released under `snowdrop` and supported in 2.1.6. The configuration therefore applies. Comment removed.